src/platform/sdl/sdl-log.c

```diff
@@ -0,0 +1,87 @@
+/* Copyright (c) 2013-2015 Jeffrey Pfau
```

Comment:
This whole file should probably be merged into `main.c`.

Reply:
Should be fixed in: https://github.com/mgba-emu/mgba/pull/2345/commits/02b165005bcfac32caf4ed53419a46ade0a6086f